Automattic/woocommerce-connect-client
# Add validation inside the add/edit packages Modal

I've added validation to the add/edit packages modal. It does the following:
1) Check if name field is empty 
2) Check if box name already exists in the list of boxes
3) requires inner dimensions and checks the format
4) if outer dimensions are provided, checks the format
5) makes sure box and max weights are numbers and not empty
6) Auto-correct spacing around "x" in the dimensions definitions

To Test:

1) Open USPS or Canada Post settings
2) Make sure you have a few boxes entered
3) Try to add a new box
4) Make sure it follows all the rules above
5) Try editing an existing box
6) Make sure it follows the rules above
7) Make sure you're able to save data after making changes
8) Experiment with variations of spacing around the "x" in the dimensions. It should be auto-corrected.

@kellychoffman @jeffstieler @allendav @jkudish

Closes #297 
Closes #306 
Closes #324
